api/include/mfxdefs.h

```diff
@@ -181,6 +181,11 @@ typedef enum
     MFX_WRN_FILTER_SKIPPED              = 10,   /* one of requested filters has been skipped */
     MFX_WRN_INCOMPATIBLE_AUDIO_PARAM    = 11,   /* incompatible audio parameters */
 
+#if MFX_VERSION >= MFX_VERSION_NEXT
+    /* low-delay partial output */
+    MFX_ERR_NONE_PARTIAL_OUTPUT         = 12,   /* frame is not ready, but bitstream contains partial output */
```

Comment:
As far as I remember we agreed not to add such status because application won't use it. Please explain what has changed?

Reply:
This one is required as we need to report status of incomplete bitstream in sync operation. You confuse it with internal status to respin scheduler.

Reply:
Okay. Thanks. So if application indeed can receive it, would you mind describing in the man how app should handle it?

Reply:
It should rerun sync operation again until it return error or MFX_ERR_NONE

Reply:
in the man..

Reply:
Agree. Pavel, please add a section to the SyncOperation documentation, describing the workflow around MFX_ERR_NONE_PARTIAL_OUTPUT return code, including some information about bitstream buffer management

Reply:
The section was added long time ago. Any update for review?